class-wc-calypso-bridge.php

```diff
@@ -74,10 +74,28 @@ function_exists( 'WC' ) &&
 	 * Constructor.
 	 */
 	public function __construct() {
+		add_action( 'muplugins_loaded', array( $this, 'deactivate_duplicate_tiktok' ), PHP_INT_MAX );
 		add_action( 'plugins_loaded', array( $this, 'initialize' ), 0 );
 		add_action( 'plugins_loaded', array( $this, 'load_translation' ) );
 	}
 
+	/**
+	 * Deactivate TikTok for WooCommerce if both TikTok for WooCommerce and Business are active (they are the same).
+	 *
+	 * @since x.x.x
+	 *
+	 */
+	public function deactivate_duplicate_tiktok() {
+		$active_plugins  = get_option( 'active_plugins' );
```

Comment:
I'd suggest using a more defensive approach here to avoid regressions:

```suggestion
		$active_plugins  = (array) get_option( 'active_plugins', array() );
```

Reply:
Thanks Chris! I like the way you think :)